roles/common/tasks/networking.yml

```diff
@@ -13,6 +13,35 @@
   with_items: network_interfaces_d
   tags: ['prereboot']
 
+- name: set slrouter address
+  set_fact: sl_router_address="{{ ansible_default_ipv4.network.split('.')[0] }}.{{ ansible_default_ipv4.network.split('.')[1] }}.{{ansible_default_ipv4.network.split('.')[2] }}.{{ (ansible_default_ipv4.network.split('.')[3] | int) + 1 }}"
+  when: ansible_default_ipv4.interface == hostvars[inventory_hostname][primary_interface].device   
+
+- name: add static route for multi subnet
+  add_sys_route: state=present
```

Comment:
The ip replace would always replace a route if it exists or add a new one if the route does not exists. However this module will only add a route if the network route does not exist. So customization (if any) to the existing routes on the remote node will be preserved even though it would be a rare case.